P5 154 introduce the three issues of mi (#102)
aau-hcl-p5/main: report/4.Design/Machine_Intelligence.tex

An MI model would then be able to predict the movement of the object.
This would allow the \texttt{F.L.A.T} to simply shoot at the predicted location, instead of continously tracking the object.
This approach would also be continously refined, if targets hit was tracked aswell.
A disadvantage to this approach is that object classification would also be required, to allow distinction of the movement of different objects.
This would allow for an iterative approach, assuming object classification and localization was implemented first, the target trajcetory could then be added on top of the

\subsection{Variable motor power}
An interesting result from the motor tests done in Section~\ref{des:sec:hwtest}, was that motors with more weight, required more power to move the same distance as a motor without added weight.
This also includes the additional force of gravity, as a motor moving downwards, required less power than a motor that moved upwards.
Additionally, the wires that connect the motors to the NXT, also added additional variable weight, depending on how loose or tense the wire was.

> This is a problem, as it could potentially impact precision, which is quite obviously important, when the goal is to hit a fast moving object.

Reviewer: ```suggestion
This is a problem, as it could potentially have an impact on precision, which is important when the goal is to hit a fast moving object.
```
